### ISSUE TITLE
[BUG] Agent is registered and tunnel is created too

### ISSUE THREAD

**Issue by 1234usama** [nicocha30/ligolo-ng#106]
I run the agent on windows machine, it got registered on my ligolong proxy server and after that i created a tunel and added the route, but i still cant able to ping and interact with any internal host. Dont know why

Ps i also run the start command to start the tunnel.

**Comment by 1234usama** [comment]
Cc

**Comment by w2xim3** [comment]
I connected the agent to the proxy and started a tunnel.

I added a route to a listener on port 4444 (with nc -lvnp 4444), and the tunnel was working correctly. However, when I created a route to a web server (using python -m http.server), the listener stopped working after 1 or 2 requests. When I tried to stop the listener with listener_stop, the message indicated that the listener's tunnel was already closed.

@nicocha30  
Ligolo-ng 0.7.2-alpha / 4ac377a17816cab1b4a807dd309efe63c7ee6e5e / 2024-08-25T17:23:01Z
